about.md

```diff
@@ -0,0 +1,145 @@
+
+* Probably move this later, but I didn't want to forget to include it in the same commit as the image file.
+* Add example images for other themes.
+
+
+* insert some images showing Pico in use.  Borrow from readme?
+  * Can the site do thumbnails and lightboxing?
+* Add a style for these images? Their lack of a border or shadow makes them hard to distinguish from the page background.
+  * Again, can these be thumbnailed and lightboxed? That might work better.
+
+* Larger "About" portion at the beginning.
+  * Should contain all the "Why's" of Pico, before the "How's"
+
+Now let's dive into what makes Pico different from other solutions.
+
+## A Flat File CMS
+
+Pico is a Content Management System, or CMS.  If you've heard the term before, it's likely in the context of WordPress, Joomla, Drupal (intentionally not linking to the competition ;P) or many of the other popular CMS's on the block.  A Content Management System makes creating a beautiful and well-organized website easy and frustration free.  It can also help those without extensive knowledge of programming on the web to achieve the same professional quality website as those that do.
+
+ * Last sentence is a bit rough....
+ * Add headers for CMS and Flat file or is that excessive?
+
+Most Content Management Systems make use of extensive databases to hold their content.  These databases can sometimes require a confusing setup process, though this is often obscured by one-click setup utilities.  While they can be simple to set up thanks to these utilities, users may experience difficulty when trying to modify, edit, or migrate their content.  Everything that makes their site "what it is" is locked up in a hard-to-manage database.  Anyone who's tried to migrate their WordPress installation from one server to another has likely experienced this pain.
+
+Pico is different.  With Pico, all your content is stored as "flat files", which is pretty much exactly as it sounds.  When using Pico, all your site content is stored as simple text files.  It remains readily available to modify, edit, organize, and migrate as you see fit.
+
+There really isn't that much more to say about it.  The beauty of Pico is in its simplicity.  If you want to create a new page, you make a new file.  That's it.  But that's far from the end of the story.  Despite Pico's simplicity, you'll find it incredibly powerful, and ready to take on any task.
+
+## Web Pages Without HTML
+
+Writing HTML is annoying.  Traditionally, if you wanted to build a web page, you had no choice but to fumble around with HTML.  Every paragraph of your site would be contained in a `<p>`paragraph tag`</p>`, links would have an `<a href="http://example.com">`anchor tag`</a>` wrapped around them, and lists would require a heck of a lot of `<li>`list item tags`</li>`, a pair around each item.  If you've written a traditional HTML page before, you know how this can be a tedious, repetitive, and error-prone process.
```

Comment:
I guess I hadn't refreshed the page because I didn't see your last comment.

The problem is that I tend to use `code` as `pay attention, this is important` and not just for actual code. :unamused:

But yeah, the idea was to emphasize what a pain it is to HTML for your content.  I like having all the extra tags in the paragraph because it encapsulates that feeling more.  It's difficult to read, which is the point.  Someone who knows HTML will stumble on reading it and go "yeah, that *is* a pain".  Someone who doesn't will hopefully go "is that really how people used to write web pages?"

---

This all makes me think of the mentality around the HTML and CSS split.  HTML was dictated to be for *structure* while CSS was for *style*.  Maybe we need another split, where Markdown is the new *content* format for the web and HTML is just the overall page structure.

Your browser would understand Markdown and when you get to your main content, you could just switch to using Markdown for that part. (kind of the reverse of how Markdown usually gets parsed now). :laughing: